src/videojs-contrib-hls.js

```diff
@@ -307,11 +307,6 @@ class HlsHandler extends Component {
     this.options_ = videojs.mergeOptions(videojs.options.hls || {}, options.hls);
     this.setOptions_();
 
-    // start playlist selection at a reasonable bandwidth for
-    // broadband internet
-    // 0.5 Mbps
-    this.bandwidth = this.options_.bandwidth || 4194304;
```

Comment:
yeah, it could have been it it's own PR but it doesn't actually change anything since the setOptions_ call above runs the same code.